src/zeit/content/cp/interfaces.py

```diff
@@ -450,6 +449,18 @@ class IRenderedArea(IArea):
     """
 
 
+class IContentQuery(zope.interface.Interface):
+    """Mechanism to retrieve content objects.
+    Used to register named adapters for the different IArea.automatic_types.
+    """
+
+    total_hits = zope.interface.Attribute(
+        'Total number of available results (only available after calling)')
```

Comment:
Ich habs mir angeschaut. Man _könnte_ es so umbauen, dass aus `__call__` ein Result-Objekt rauskommt, das eine Liste ist und `total_hits` mit draufstehen hat. Das wird aber in der Benutzung im Friedbert eher unlesbarer als besser (weil noch andere Sachen vom Query-Objekt runtergelesen werden, und weil man dann in zeit.web mitten in der `values()`-Methode mit dem Query-Result noch was machen müsste d.h. nen Extensionpoint bräuchte etc.pp.). Insofern würde ich es glaub ich erstmal so lassen. :-/